src/util/sss_ptr_hash.c

```diff
@@ -332,7 +332,7 @@ void sss_ptr_hash_delete_all(hash_table_t *table,
 {
     struct sss_ptr_hash_value *value;
     hash_value_t *values;
-    size_t count;
+    unsigned long int count;
     size_t i;
     int hret;
```

Comment:
Good point, in theory this might cause an infinite loop if count > 2^32. Do you think this should be fixed in the other places where hash_values() is used as well? Please open a ticket in that case so it won't get lost during the holidays.